cumulusci.yml

```diff
@@ -77,11 +77,16 @@ tasks:
             path: unpackaged/config/trial
             namespace_tokenize: $project_config.project__package__namespace
 
-    add_picklist_values:
+    add_application_type_picklist_values:
         description: Adds picklist values to the given field, if they don't already exist.
```

Comment:
NIT: the description doesn't mention that it is only adding the picklist values for the application type in EDA. We may want to be specific. Admittedly the task's name itself says what field, but we may just want to be clear.